zerver/actions/create_realm.py

```diff
@@ -242,6 +245,12 @@ def do_create_realm(
             realm=realm,
             event_type=RealmAuditLog.REALM_CREATED,
             event_time=realm.date_created,
+            extra_data={
+                "how_did_you_find_zulip": how_did_you_find_zulip,
+                # Only motivation to store these separately is to avoid displaying this on /activity page.
+                "how_did_you_find_zulip_other_description": how_did_you_find_zulip_other_description,
+                "how_did_you_find_zulip_where_ad": how_did_you_find_zulip_where_ad,
```

Comment:
Hmm, I think the `how_did_you_find_zulip_other_description` / `how_did_you_find_zulip_where_ad` could maybe be stored in the same key, since the value of `how_did_you_find_zulip` would tell you what the user submitted?

(I assume the comment about it being a separate field is just about not squashing it with the main `how_did_you_find_zulip`).

Reply:
Done.